proof/refine/AARCH64/Syscall_R.thy

```diff
@@ -349,11 +349,22 @@ lemma threadSet_tcbDomain_update_sch_act_wf[wp]:
    apply (auto simp: obj_at'_def)
   done
 
+lemma vcpuFlushIfCur_corres[corres]:
+  "corres dc (pspace_aligned and pspace_distinct and valid_arch_state and tcb_at tptr)
+             (pspace_aligned' and pspace_distinct' and no_0_obj')
+             (vcpu_flush_if_current tptr) (vcpuFlushIfCur tptr)"
+  unfolding vcpu_flush_if_current_def vcpuFlushIfCur_def
+  by (corres wp: arch_thread_get_wp archThreadGet_wp)
+
+crunch vcpu_flush_if_current
```

Comment:
ah yes, seeing `vcpu_flush_if_current` and `vcpuFlushIfCur` side by side, I definitely think the latter should be `vcpuFlushIfCurrent`

Reply:
Confusingly, I went with `vcpu_flush_if_current` in the C as well so I'm not sure why I did the Haskell differently. Will change.